src/Google/GlobalSiteTag.php

```diff
@@ -133,18 +173,14 @@ function gtag(){dataLayer.push(arguments);}
 	 *
 	 * @param string $ads_conversion_id Google Ads account conversion ID.
 	 * @param string $ads_conversion_label Google Ads conversion label.
+	 * @param int    $order_id The order id.
 	 */
-	public function maybe_display_event_snippet( string $ads_conversion_id, string $ads_conversion_label ): void {
+	public function maybe_display_event_snippet( string $ads_conversion_id, string $ads_conversion_label, int $order_id ): void {
```

Comment:
Since we've added multiple event snippets I don't think this function name/purpose is very clear anymore. Can we rename this function to describe which event snippet(s) it's displaying.

Also since we now have a group name 'GLA' is it still relevant to pass the conversion ID and label for tracking? Or can we just send the event to the same group as the other events?

Reply:
Updated the name. 'GLA' group is relevant to passing the conversion ID but I'm not sure if it is same as $ads_conversion_id/$ads_conversion_label. Could I know how to make sure if they are the same group?

Reply:
In [this document](https://support.google.com/google-ads/answer/9028614?hl=en#zippy=%2Cinstall-with-global-site-tag) it seems to indicate that the purchase event should also be sent with `conversion_id/label`. Which is why I asked if they should all have a matching `send_to`. I wasn't able to find any clear details in the docs though. Is there someone from the Google Ads team that could clarify this?

Reply:
Thanks for pointing this out. After checked [this doc](https://docs.google.com/presentation/d/1JVJ7wsA22nwB-YWOJGAqxcouIDsv7D9cPj6UjXK0eBI/edit?userstoinvite=chennyx%40google.com&resourcekey=0-A2hcqh8CwxpRGQGQLPVk6A#slide=id.g10e1beef46c_0_373), it should use $ads_conversion_id/$ads_conversion_label here as you said. Updated.